features/04_aruba_api/core/expand_path.feature

```diff
@@ -69,7 +69,7 @@ Feature: Expand paths with aruba
     RSpec.describe 'Expand path', :type => :aruba do
       let(:path) { '~/path/to/dir' }
 
-      it { expect(expand_path(path)).to match %r</home/[^/]+/path/to/dir> }
+      it { expect(expand_path(path)).to match %r</home/([^/]+/)+path/to/dir> }
```

Comment:
I think the intent here was for `[^/]+` to match exacly a user name.

The correct fix is probably to remove the 'before 1.0.0' part of the feature and fix the 1.0.0 part to not set things up in the `before` section.